crates/vertigo/src/lib.rs

```diff
@@ -132,7 +134,7 @@ pub use instant::{
 };
 pub use websocket::{WebsocketConnection, WebsocketMessage};
 
-pub use vertigo_macro::include_static;
+pub use vertigo_macro::{include_static, bind};
```

Comment:
Make it:

```
/// Allows to include a static file
///
/// This will place the file along with the rest of generated files. The macro returns a public path to the file with it's hash in name.
pub use vertigo_macro::include_static;

/// Allows to create an event handler based on provided arguments
pub use vertigo_macro::bind;
```